doc/api/modules.md

```diff
@@ -841,11 +841,19 @@ added: v9.3.0
 A list of  the names of all modules provided by Node.js. Can be used to verify
 if a module is maintained by a third-party module or not.
 
+Note that `module` in this context isn't the same object that's provided
+by the [module wrapper][]. To access it require the `Module` module:
```

Comment:
I don't believe we typically capitalize module names so I suspect this should be ``` the `module` module ```?

Reply:
Regardless, please put a comma beofre `require the`.

Reply:
Whoops, ignore my comment on casing (`module` vs. `Module`) since, you know, that's the whole point... 🙃  Sorry.

Reply:
Comma added 👍